index.html

```diff
@@ -205,34 +205,37 @@ <h2>Frequently Asked Questions</h2>
     </section>
 
     <!-- Contact -->
+    <section id="contact">
+        <div id="contact-content">
+            <h2>Contact</h2>
+            <p>Feel free to let us know of any questions, comments, and/or concerns at <a href="mailto:hello@brickhack.io">hello@brickhack.io</a></p>
+            <p>Interested in joining our team? <a href="./club.html">Check out our club</a></p>
+        </div>
+        <div id="contact-socials">
+            <a href="https://twitter.com/brickhackrit" target="_blank">
+                <i class="fab fa-twitter"></i>
+            </a>
+            <a href="https://www.facebook.com/brickhackrit" target="_blank">
+                <i class="fab fa-facebook"></i>
+            </a>
+            <a href="https://www.instagram.com/brickhack/" target="_blank">
+                <i class="fab fa-instagram"></i>
+            </a>
+            <a href="https://snapchat.com/add/brickhack" target="_blank">
+                <i class="fab fa-snapchat"></i>
+            </a>
+        </div>
+    </section>
+
+    <!-- Footer -->
     <footer>
```

Comment:
Footer content isn't vertically centered on desktop:

![image](https://user-images.githubusercontent.com/16471311/144720123-2b8045fb-e35f-4267-bac9-b4f5d37295cd.png)

Playing around in the web inspector, this seems to fix it:
- Remove vertical padding on `<p>`s (could remove padding entirely and use `gap` instead for mobile spacing)
- `align-items: center` on `footer` and `#footer-right` (but set it back to default on mobile for when `#footer-right` becomes a column)

Reply:
I fixed it in next commit, just had some odd padding settings in there